CHANGELOG.md

```diff
@@ -1,21 +1,39 @@
 # App Center SDK for .NET Change Log
 
-## Version 4.0.1 (under development)
+## Version 4.1.0
 
 ### AppCenter
 
 * **[Feature]** Add a `SetMaxStorageSizeAsync` API which allows setting a maximum size limit on the local SQLite storage. The default value is 10MiB.
 
+#### iOS
+
```

Comment:
please move it to [here](https://github.com/microsoft/appcenter-sdk-dotnet/pull/1484/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR15)